action.yml

```diff
@@ -114,6 +125,20 @@ runs:
         registry-url: 'https://npm.pkg.github.com'
         scope: '@searchspring'
 
+    - name: Authenticate siteId and secretKey
+      id: authenticate
+      shell: bash
+      run: |
+        npm install
```

Comment:
Should NPM installation come after the loading of cached modules?

Also, if possible, I think we should avoid installing `snapfu` unless it is being used for the job.